### PR TITLE
Store pkeys in store

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Store pkeys in store permanently, so that there is no need to create temporary files and keep track of deleting them.

### CODE REVIEW

File: run_service.sh [hunk 1]
```diff
@@ -187,63 +187,60 @@ get_on_chain_service_state() {
 store=".trader_runner"
 rpc_path="$store/rpc.txt"
 operator_keys_file="$store/operator_keys.json"
+operator_pkey_path="$store/operator_pkey.txt"
 keys_json="keys.json"
 keys_json_path="$store/$keys_json"
+agent_pkey_path="$store/agent_pkey.txt"
```
Comment (Adamantios) [inline run_service.sh:193]: This will make all the participants' trader runner stores corrupted!

**Review by Adamantios — CHANGES_REQUESTED** [review]
I fixed the service id issue. Please address this comment:

https://github.com/valory-xyz/trader-quickstart/pull/54#discussion_r1365153249

File: run_service.sh (hunk 1 above)
Comment (jmoreira-valory) [reply run_service.sh:193]: Can we fix it with a "patch" before line 243 as follows:

if operator_keys_file exists --> create operator_pkey.txt
if keys_json exists --> create agent_pkey.txt

(create using same approach as in create_storage)

Comment (Adamantios) [reply run_service.sh:193]: Yes, sounds like a good idea. This will make it backwards compatible.

File: run_service.sh [hunk 2]
```diff
@@ -187,69 +190,77 @@ get_on_chain_service_state() {
...
 }
 
 # Function to read and load the .trader_runner storage information if it exists.
 # Also sets `first_run` flag to identify whether we are running the script for the first time.
 try_read_storage() {
     if [ -d $store ]; then
+
+        # INFO: This is a fix to avoid corrupting already-created stores
+        if [[ -f "$operator_keys_file" && ! -f "$operator_pkey_path" ]]; then
+            echo "1"
```
Comment (Adamantios) [inline run_service.sh:246]:
```suggestion
```

File: run_service.sh [hunk 3]
```diff
@@ -187,69 +190,77 @@ get_on_chain_service_state() {
...
+            operator_pkey=$(get_private_key "$operator_keys_file")
+            echo -n "$operator_pkey" > "$operator_pkey_path"
+        fi
+
+        # INFO: This is a fix to avoid corrupting already-created stores
+        if [[ -f "$keys_json_path" && ! -f "$agent_pkey_path" ]]; then
+            echo "2"
```
Comment (Adamantios) [inline run_service.sh:253]:
```suggestion
```

**Review by Adamantios — APPROVED** [review]

**Pull request merged**